furusiya/model/components/xp.py

```diff
@@ -0,0 +1,27 @@
+from main_interface import message
+from model.components.base import Component
+from model.config import config
+from model.systems.fighter_system import FighterSystem
+
+
+class XP(Component):
+    def __init__(self, owner, level=1, xp=0):
+        super().__init__(owner)
+        self.level = level
+        self.xp = xp
+        self.stats_points = 0
+
+    def _xp_next_level(self):
+        return 2 ** (self.level + 1) * config.data.player.expRequiredBase
```

Comment:
This is too specific to the user......   Hmm. The simplest solution is to pass in `expRequiredBase` as a value to `init`. Alternatively -- and this is a really over-engineered solution -- we can pass in a lambda instead.

But don't. I'm watching you!